addons/website_mail_channel/static/src/js/website_mail_channel.snippet.js

```diff
@@ -117,6 +117,8 @@ publicWidget.registry.follow_alias = publicWidget.Widget.extend({
             else {
                 self.$target.find('.js_mg_email').addClass('d-none');
             }
+        }).guardedCatch(function(res) { // case you don't have read access on record (modelConverter crash)
```

Comment:
space between `function` and parenthesis. Also we can now do ES6 ! `}).guardedCatch((res) => {`